naas/models/campaigns.py

```diff
@@ -0,0 +1,128 @@
+import naas
+from naas.configuration import Configuration
+from naas.errors import InvalidRequestError, RecordNotFoundError
+from naas.models.campaign import Campaign
+from naas.models.error import Error
+
+
+class Campaigns(object):
+    """
+
+    Campaigns
+    ===============
+
+    This returns an instance of the campaigns model
+    """
+
+    def __init__(self, collection):
+        self.collection = list(collection)
+        self.index = len(self.collection)
+
+    def __iter__(self):
+        """ Implement iterator """
+        return map(lambda r: Campaign(r), self.collection)
+
+    def next(self):
+        if self.index == 0:
+            raise StopIteration
+        self.index = self.index - 1
+        return self.collection[self.index]
+
+    @classmethod
+    def list_by_project_id(project_id, params=None):
+        """
+        Helper method to retrieve from the request
+        :param project_id: str
+        :param params: dict
+        :return: Campaigns
+        """
+        if params is None:
+            params = {}
+
+        request = naas.requests.Campaigns.list_by_project_id(
```

Comment:
@arthuroe - This returns `AttributeError: module 'naas.requests' has no attribute 'Campaigns'`